assemblies/assembly-designing-and-building-cases.adoc

```diff
@@ -36,7 +36,7 @@ include::{jbpm-dir}/DesignBuildCases/case-management-adhoc-con.adoc[leveloffset=
 include::{jbpm-dir}/DesignBuildCases/case-management-adding-dynamic-tasks-using-API-proc.adoc[leveloffset=+1]
 include::{jbpm-dir}/DesignBuildCases/case-management-dynamic-user-task-API-proc.adoc[leveloffset=+2]
 include::{jbpm-dir}/DesignBuildCases/case-management-dynamic-service-task-API-proc.adoc[leveloffset=+2]
-include::{jbpm-dir}/DesignBuildCases/case-management-dynamic-subprocess-API-proc.adoc[leveloffset=+2]
+include::{jbpm-dir}/DesignBuildCases/case-management-dynamic-sub-process-API-proc.adoc[leveloffset=+2]
```

Comment:
Is this correct?  I don't see the file itself being renamed in this PR?

Reply:
I corrected 3 includes to match the original file names that do not contain hyphens.